deploy/memcache-dep.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-# You can optionally deploy memcache, for the Flux daemon to cache
+# The memcached deployment is used by Flux to cache
```

Comment:
This does not seem to be a related change.

Reply:
Because of that "optionally" we've got users on Slack that thought Flux will work without memcached. But even with `--registry-exclude-image=*` memcached is mandatory.

Reply:
I'm not arguing that the sentence is correct, just that you should keep the PR on point.